theseus_gui/src/pages/instance/Options.vue

```diff
@@ -298,55 +298,123 @@
       />
     </div>
   </Card>
-  <Card>
+  <Card v-if="instance.metadata.linked_data">
     <div class="label">
       <h3>
-        <span class="label__title size-card-header">Instance management</span>
+        <span class="label__title size-card-header">Modpack</span>
       </h3>
     </div>
-    <div v-if="instance.metadata.linked_data" class="adjacent-input">
+    <div class="adjacent-input">
+      <label for="general-modpack-info">
+        <span class="label__description">
+          <strong>Modpack: </strong> {{ instance.metadata.name }}
+        </span>
+        <span class="label__description">
+          <strong>Version: </strong>
+          {{
+            installedVersionData.name.charAt(0).toUpperCase() + installedVersionData.name.slice(1)
+          }}
+        </span>
+      </label>
+    </div>
+    <div v-if="!instance.locked" class="adjacent-input">
+      <Card class="unlocked-instance">
+        This is an unlocked instance. There may be unexpected behaviour unintended by the modpack
+        creator.
+      </Card>
+    </div>
+    <div v-else class="adjacent-input">
+      <label for="repair-profile">
+        <span class="label__title">Unlock instance</span>
+        <span class="label__description">
+          Allows modifications to the instance, which allows you to add projects to the modpack. The
+          pack will remain linked, and you can still change versions. Only mods listed in the
+          modpack will be modified on version changes.
+        </span>
+      </label>
+      <Button id="repair-profile" @click="unlockProfile"> <XIcon /> Unlock </Button>
```

Comment:
this is a destructive action- it should have a confrim modal

Reply:
should also just use the unlock icon lol

Reply:
couldnt find an unlock icon in theseus or omorphia so using the lock one instead